djorg/djorg/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
```

Comment:
It looks like you have a project named djorg nested inside another project named djorg, so there are two different sets of  config files and no models?